cardano-testnet/src/Testnet/Property/Utils.hs

```diff
@@ -1,17 +1,25 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE MonoLocalBinds #-}
 {-# LANGUAGE NumericUnderscores #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 
 module Testnet.Property.Utils
   ( integration
   , integrationRetryWorkspace
   , integrationWorkspace
   , isLinux
+  , runInBackground
 
   , QueryTipOutput(..)
   , queryTip
-  , waitUntilEpoch
   , queryUtxos
+  , waitUntilEpoch
+  , getEpochStateView
+  , findAllUtxos
+  , findUtxosWithAddress
+  , findLargestUtxoWithAddress
```

Comment:
I think all these functions would be better placed in a sibling `Query.hs` module. Those functions are all used in the middle of tests (in the business logic) to obtain some data. I think that being in the `Utils.hs` file doesn't indicate how central those functions are to most integration tests.

Reply:
Moved to Query.hs